labs/lab09.md

```diff
@@ -0,0 +1,35 @@
+# Lab 8: Introduction to React.js
+
+React is a declarative, efficient, and flexible JavaScript library for building interactive and dynamic user interfaces. It lets you compose complex UIs from small and isolated pieces of code called “components”.In this lab, students will learn the fundamentals of React.js and create an interactive quiz application.
+
+## Deliverables
+- [ ] Modify the implementation to retrieves questions and answer options from an external quizData file and correctly displays them in the quiz interface.
+- [ ] Enhance the user interface to indicate the selected answer, providing a more visually appealing user experience
+- [ ] Extend the Quiz component to record user choices and display the total score of the selected options when the "Submit" button is pressed
+
+## Instructions
+Clone the Quiz App repository from: https://github.com/CMU-17-214/f23-rec09 and run
+
+```
+npm install
+npm start
+```
+![Local Image](src/image/starterPic.png)
+
+This will start the front-end server at http://localhost:3000. You can update the front-end code as the server is running in the development mode (i.e., npm start). It will automatically recompile and reload.
```

Comment:
```suggestion
This will start the front-end server at http://localhost:3000. You can update the front-end code as the server is running in the development mode (i.e., `npm start`). It will automatically recompile and reload.
```